src/const.js

```diff
@@ -0,0 +1,17 @@
+export const ActivityTypeEnum = {
+  BOOK: 0,
```

Comment:
Isn't book too specific ? This type is more about a content, like Book, Video, Article, .. ? 
I propose CONTENT instead of BOOK.